application/src/main/scala/com/azavea/franklin/api/endpoints/SearchEndpoints.scala

```diff
@@ -30,14 +30,17 @@ object SearchEndpoints {
             Option[String]
         )) => {
           val (temporalExtent, bbox, collections, ids, limit, next) = tup
+          // query is empty here because entering query extension fields in url params is
```

Comment:
urlencoding helps; but you can use postman without explicit urlencoding at it works fine.

Reply:
it's not really a question of how to do it as much as whether it's a sort of insane thing to do -- the queries start out as nested json, and I'd like to discourage people from trying to use the API that way, since we already have a `POST` query endpoint that accepts more complicated params

Reply:
ooooh :+1: